tests/test_integration.py

```diff
@@ -42,8 +42,12 @@ async def create_channel(src, dest, funding: int):
     finally:
         assert await src["api"].close_channel(channel)
         await asyncio.wait_for(check_channel_status(src, dest, status="PendingToClose"), 10.0)
+
+        # need to wait some more time until closure time has passed
+        await asyncio.sleep(10)
+
```

Comment:
This is what the next wait_for is for... let's not use raw sleeps, but checking sleeps.

Reply:
The advantage of using checked wait_fors is that even if the value looks huge, like 30.0s, it will get to that only if the closure really fails, otherwise it might even end earlier and is future proof, if the timeout were to change.

Reply:
OK, now I get it. Why did it work in my setup?